src/pseudo/NormConservingPsp.jl

```diff
@@ -97,6 +105,44 @@ function eval_psp_energy_correction end
 eval_psp_energy_correction(psp::NormConservingPsp, n_electrons) =
     eval_psp_energy_correction(Float64, psp, n_electrons)
 
+"""
+    eval_psp_density_valence_real(psp, r)
+
+Evaluate the atomic valence charge density in real space.
+"""
+eval_psp_density_valence_real(psp::NormConservingPsp, r::AbstractVector) = 
+    eval_psp_density_valence_real(psp, norm(r))
```

Comment:
Ok, let's discuss this further when we get to the PseudoPotentialIO PR. I agree that marker structs are only worth it if you can do something with them (i.e. maybe pass them through layers or expose to a user etc.). Otherwise it's essentially the same thing as having multiple functions.